tag-rule/nitagrule.yml

```diff
@@ -344,7 +353,7 @@ paths:
         401:
           description: Unauthorized
           schema:
-            $ref: '#/definitions/Error'
+            $ref: '#/responses/Unauthorized'
```

Comment:
Responses refs have to go one more level up, not under a schema. Applies to below places as well.

Reply:
They should all be cleaned up now